test/Concurrency/sendable_checking.swift

```diff
@@ -113,3 +113,106 @@ class SendableSubclass: NSClass, @unchecked Sendable { }
 func testSubclassing(obj: SendableSubclass) async {
   acceptCV(obj) // okay!
 }
+
+
+@available(SwiftStdlib 5.1, *)
+protocol P {
+  func foo (x : @Sendable () -> ()) async -> ()
+
+  func bar(x : () -> ()) async -> ()
```

Comment:
I actually wonder if the warning can be emitted always already when we see such func in a protocol on an async func? Since an async func, in a class/struct will execute on global pool, and on an actor "on the actor", so in any case this will cross an async boundary so needs to be sendable always, doesn't it?

Reply:
methods of classes can also be `async`, so this protocol could also be used to allow one class's async method to call another class's async method without crossing an isolation boundary. This I believe is a valid use case for a protocol requirement for an async method without sendable parameters.